### ISSUE TITLE
error from converting function reference with quarto_website

### ISSUE THREAD

**Issue by benzipperer** [etiennebacher/altdoc#179]
Hi, I'm trying to convert an existing package from pkgdown to altdocs. I get stuck on `altdocs::render_docs()`. Do you have any suggestions?

```r
> altdoc::render_docs()
ℹ Adding ^_quarto$ to .gitignore

── Basic files ───────────────────────────────────────────────────────────────────
✔ NEWS imported.
✔ LICENSE imported.
✔ README imported.

── Man pages ─────────────────────────────────────────────────────────────────────
ℹ Found 11 man pages to convert.
Error in file(con, "r") : cannot open the connection
In addition: Warning message:
In file(con, "r") :
  cannot open file '/home/benzipperer/realtalk/_quarto/docs/man/available_price_indexes.md': No such file or directory
✖ Converting function reference 1/11: available_price_indexes [424ms]
```

To reproduce this, clone the realtalk repo
```
git clone https://github.com/Economic/realtalk
```
and then in that directory in R
```r
altdoc::setup_docs(tool = "quarto_website")
altdoc::render_docs()
```

Thanks for any advice!

**Comment by etiennebacher** [comment]
Hi, I can reproduce this error. It looks like it only happens with `quarto_website`, not with `docute` for example. 

@vincentarelbundock do you want to deal with this since you made all the quarto integration?

**Comment by vincentarelbundock** [comment]
@benzipperer `altdoc` is in the middle of a complete overhaul, and it is not *quite* ready for public consumption yet. But thanks anyway for raising this issue, because it's going to help me diagnose a lot of problems.

I don't have a lot of time now, but here are some issues I see immediately:

+ [ ] Quarto output probably requires a README.qmd file. We should either raise an informative error for the user or modify the code to work with README.md or README.Rmd.
+ [ ] The link insertion code that @etiennebacher wrote only works on `.md` files, which raises an error. In PR https://github.com/etiennebacher/altdoc/pull/180 I skip this part of the code, but there must be an elegant solution.
+ [ ] Even with these two things corrected, I still see a weird problem:


```bash
ERROR: NotFound: No such file or directory (os error 2): rename '/home/vincent/repos/realtalk/_quarto/index.html' -> '/home/vincent/repos/realtalk/_quarto/_site/index.html'

Stack trace:
    at Object.renameSync (ext:deno_fs/30_fs.js:215:7)
    at renderProject (file:///opt/quarto/bin/quarto.js:76560:22)
    at eventLoopTick (ext:core/01_core.js:181:11)
    at async Command.fn (file:///opt/quarto/bin/quarto.js:80606:32)
    at async Command.execute (file:///opt/quarto/bin/quarto.js:8111:13)
    at async quarto (file:///opt/quarto/bin/quarto.js:111318:5)
    at async file:///opt/quarto/bin/quarto.js:111336:9
```

**Comment by etiennebacher** [comment]
> The link insertion code that @etiennebacher wrote only works on .md files, which raises an error. In PR https://github.com/etiennebacher/altdoc/pull/180 I skip this part of the code, but there must be an elegant solution.

If that was the case, it should also error with other docs generators, no? `altdoc::render_docs("docute")` works fine for me

**Comment by vincentarelbundock** [comment]
No because contrary to all the other frameworks, `quarto_render()` has to be called near the very end of the process. That is, *after* you insert the links. When you do insert those links, you look for `.md` files only, but at that point we only have `.qmd` files in the folder. So we get a "No such file" error.

**Comment by vincentarelbundock** [comment]
@benzipperer I think this should be resolved in version 0.2.2.9012 from Github. I also opened a PR in your repo to illustrate what this could look like and to highlight a couple necessary tweaks: https://github.com/Economic/realtalk/pull/5

**Comment by vincentarelbundock** [comment]
If anybody is curious, this is what the website looks like: https://arelbundock.com/realtalk/
